Backend/Backend/Services/PodcastService.cs

```diff
@@ -24,7 +27,7 @@ public class PodcastService : IPodcastService
         {
 
 
-            if (!AllowedTypes.Contains(createPodcastRequest.coverImage!.ContentType))
+            if (createPodcastRequest.coverImage != null && !AllowedTypes.Contains(createPodcastRequest.coverImage!.ContentType))
```

Comment:
inverted the null condition here again

Reply:
Same as before. copying this comment to show it:

you can disregard this. you are correct here. Since we want the file to both exist AND be the wrong format, this is correct. Add a comment however to show it, since the logic is kinda confusing